language/fixtures/break.rb

```diff
@@ -128,6 +128,19 @@ def break_in_block_in_while
         note :c
       end
     end
+
+    def break_in_another_thread
+      note :a
+      thread = Thread.new do
+        note :b
+        break :break
+        note :c
+      end
+      thread.join rescue nil
+      note :d
+      thread.value
+      note :e
```

Comment:
How about returning the `thread` and in the spec above do the `thread.value` (inside the `should raise_error`)?

BTW, I'm not particularly fond of fixtures, so if it makes it easier to move the whole code to the spec I'm fine with that :smiley: